SecurityManager/Clients.Web.Test/Global.asax.cs

```diff
@@ -47,12 +47,24 @@ protected void Application_Start (object sender, EventArgs e)
           defaultServiceLocator.GetInstance<IPrincipalProvider>() is SecurityManagerPrincipalProvider,
           "Wrong IPrincipalProvider is configured");
 
+#if DEBUG
+      const string configuration = "Debug";
+#else
+      const string configuration = "Release";
+#endif
+
       _resourceVirtualPathProvider = new ResourceVirtualPathProvider (
           new[]
           {
-              new ResourcePathMapping ("Remotion.Web", @"..\..\Remotion\Web\Core\res"),
+              new ResourcePathMapping ("Remotion.Web/Html/jquery-1.6.4.js", @"..\..\Web\Core\res\Html\jquery-1.6.4.js"),
```

Comment:
Probably should be a dedicated cleanup PR, but I have an idea how to remove the code duplication. Please create a task and we do this in the next sprint.